config/config.php

```diff
@@ -229,4 +229,5 @@
     $services->set(\PHPStan\PhpDocParser\Lexer\Lexer::class);
     $services->set(TypeParser::class);
     $services->set(ConstExprParser::class);
+    $services->set(\Rector\Core\Util\FileHasher::class);
```

Comment:
It not yet removed?

Reply:
I think this line is required to allow injection the service into the other classes via __construct()?